include/zrepl_prot.h

```diff
@@ -39,7 +39,8 @@ extern "C" {
 
 #define	MIN_SUPPORTED_REPLICA_VERSION	3
 
-#define	REPLICA_VERSION	4
+#define	REPLICA_REBUILD_VERSION	5
```

Comment:
different name that helps to know the feature in rebuild..
for this case, like, MIN_RESIZE_REPLICA_VERSION

Reply:
might be MIN_RESIZE_REBUILD_VERSION

Reply:
or RESIZE_REBUILD_MIN_VERSION?

Reply:
the fix has RESIZE op code and AFS_STARTED op code, so it should not say RESIZE specifically

Reply:
on the other hand I think RESIZE_REBUILD_MIN_VERSION seems to best fit as AFS started is to fix a bug but RESIZE is a major feature. so going ahead with RESIZE_REBUILD_MIN_VERSION.